config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -336,6 +335,9 @@ ext:samenvattingPropertyGroup
 ext:kwaliteitAndUitvoeringPropertyGroup
     a               form:PropertyGroup ;
     sh:name         "Kwaliteit & uitvoering" ;
+    # The title appears a bit larger in the Miro, so in theory all property groups below this one should be nested but that adds a border and requires some refactoring which we don't really have time for anymore.
+    # We use the skin option as a workaround to make the title look larger.
+    form:options    """{ "skin": 2 }""";
```

Comment:
Alternatively we could also configure the level of the other property groups which might be better.